### PR TITLE
sponsors: Add intro text and more options.

### PR DESCRIPTION
**Pull request by laurensvalk**

### CODE REVIEW

File: src/sponsor/SponsorDialog.tsx [hunk 1]
```diff
@@ -25,6 +28,30 @@ const SponsorDialog: React.VoidFunctionComponent = () => {
             icon="heart"
         >
             <div className={classNames(Classes.DIALOG_BODY, Classes.RUNNING_TEXT)}>
+                <h4>{i18n.translate('whoAreWe.heading')}</h4>
+                <p>
+                    {i18n.translate('whoAreWe.team')}&nbsp;
```
Comment (dlech) [inline src/sponsor/SponsorDialog.tsx:33]: Probably don't need a non-breaking space here. A regular space is fine.
```suggestion
                    {i18n.translate('whoAreWe.team')}{' '}
```

Comment (dlech) [reply src/sponsor/SponsorDialog.tsx:33]: Or if we ever support RTL languages, the link really should be filling in a placeholder in the translation.

**Comment by dlech** [comment]
> EDIT: exporting as plain SVG almost works.

For the other icons, I've been using *Save As... Optimized SVG* in Inkscape.

**Comment by laurensvalk** [comment]
Thanks. Updated.

File: src/sponsor/SponsorDialog.tsx (hunk 1 above)
Comment (laurensvalk) [reply src/sponsor/SponsorDialog.tsx]: I went with `{' '}`.

If filling it in, I would have left out `Learn more` and just hyperlinked `team`, but couldn't get that to work right now. Seems OK for now though.

Comment (dlech) [reply src/sponsor/SponsorDialog.tsx]: for future reference, if you ever need to do inline styling, etc. in a translated string, you can use a placeholder like this: https://github.com/pybricks/pybricks-code/pull/1295/commits/ff777ef8352e3b63a7e4c6a4b3d664b2c689a656

**Pull request merged**